### ISSUE TITLE
`buildName` doesn't pass `--build-name` to the command

### ISSUE THREAD

**Issue by Zazo032** [hey24sheep/azure-flutter-tasks#96]
Build task:
```yml
        - task: FlutterBuild@0
          displayName: 'Build Web'
          inputs:
            buildName: '$(versionName)'
            dartDefine: 'app.flavor=qa'
            projectDirectory: '.'
            target: 'web'
```

Task output:
```
Starting: Build Web
==============================================================================
Task         : Flutter Build Task
Description  : Build a Flutter application project.
Version      : 0.3.9
Author       : Hey24sheep
Help         : [More Information](https://github.com/hey24sheep/azure-flutter-tasks)
==============================================================================
/opt/hostedtoolcache/flutter/bin/flutter build web --dart-define=app.flavor=qa
```

**Comment by hey24sheep** [comment]
Hi, thanks for reporting the bug. 

Give me some time, I will fix it within a few days.

Thanks

**Comment by hey24sheep** [comment]
This issue is fixed in latest version of `0.3.16`. I am closing this issue in respect to that, if you still face issues please feel free to create a new or reopen this.
